egs/conferencing_speech_2021/local/download.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+storage_dir=
+
+. utils/parse_options.sh
+
+
+mkdir -p "$storage_dir/source_datasets"
+mkdir -p "$storage_dir/noise_datasets"
+mkdir -p "$storage_dir/linear"
+mkdir -p "$storage_dir/circle"
+mkdir -p "$storage_dir/non_uniform"
+mkdir -p data
+
+# You need to download the challenge data from uptobox it contains 3 folders.
+# Try to reproduce the same architecture of folders as uptobox ex: unzip Training_set.zip -d Training_set
+# Put the the path to these folders here
+train_set=
+dev_set=
+
+aishell_1=$storage_dir/source_datasets/aishell_1/
+aishell_3=$storage_dir/source_datasets/aishell_3/
+vctk=$storage_dir/source_datasets/vctk/
+librispeech=$storage_dir/source_datasets/librispeech_360/
+musan=$storage_dir/noise_datasets/musan/
+audioset=$storage_dir/noise_datasets/audioset/
+challenge_data=$storage_dir/challenge_data/
+
+
+# Exit on error
+set -e
+set -o pipefail
+
+
+if ! [[ -x "$(command -v youtube-dl)" ]] ; then
+  echo "This recipe requires youtube-dl for downloading Audioset. Exiting. You can run conda install -c conda-forge youtube-dl"
+  exit 1
+fi
+
+if ! test -e $aishell_1; then
+    echo "Downloading aishell-1 into $aishell_1"
+    wget -c --tries=0 --read-timeout=20 https://www.openslr.org/resources/33/data_aishell.tgz -P $aishell_1
+	  tar -xzf $aishell_1/data_aishell.tgz -C $aishell_1
+    for f in $aishell_1/data_aishell/wav/*.tar.gz; do tar xf "$f" -C $aishell_1/data_aishell/wav/; rm -rf "$f" ;done
+	  rm -rf $aishell_1/data_aishell.tgz
+    wget -c --tries=0 --read-timeout=20 https://www.openslr.org/resources/33/resource_aishell.tgz -P $aishell_1
+          tar -xzf $aishell_1/resource_aishell.tgz -C $aishell_1
+          rm -rf $aishell_1/resource_aishell.tgz
+fi
+
+
+if ! test -e $aishell_3; then
+    echo "Downloading aishell-3 into $aishell_3"
+    wget -c --tries=0 --read-timeout=20 https://www.openslr.org/resources/93/data_aishell3.tgz -P $aishell_3
+          tar -xzf $aishell_3/data_aishell3.tgz -C $aishell_3
+          rm -rf $aishell_3/data_aishell3.tgz
+
+fi
+
+if ! test -e $librispeech; then
+    echo "Downloading LibriSpeech/train-clean-360 into $librispeech"
+    wget -c --tries=0 --read-timeout=20 http://www.openslr.org/resources/12/train-clean-360.tar.gz -P $librispeech
+    tar -xzf $librispeech/train-clean-360.tar.gz -C $librispeech
+    rm -rf $librispeech/LibriSpeech/train-clean-360.tar.gz
+fi
+
+if ! test -e $vctk; then
+    echo "Downloading VCTK into $vctk"
+          wget -c --tries=0 --read-timeout=20 https://datashare.ed.ac.uk/download/DS_10283_3443.zip -P $vctk --no-check-certificate
+          unzip -q $vctk/DS_10283_3443.zip -d $vctk
+          unzip -q $vctk/VCTK-Corpus-0.92.zip -d $vctk
+          rm -rf $vctk/DS_10283_3443.zip
+          rm -rf $vctk/VCTK-Corpus-0.92.zip
+fi
```

Comment:
I think you may need to download VCTK-Corpus v0.80 instead. See the discussion here: https://github.com/ConferencingSpeech/ConferencingSpeech2021/issues/14.

Reply:
Thank you for noticing this !